domain/src/main/java/com/bottlerocketstudios/brarchitecture/domain/models/CredentialModel.kt

```diff
@@ -12,5 +13,5 @@ data class CredentialModel(val id: ProtectedProperty<String>, val password: Prot
 private const val MINIMUM_ID_LENGTH = 3
 private const val MINIMUM_PASSWORD_LENGTH = 8
 
-@JsonClass(generateAdapter = true)
+// @JsonClass(generateAdapter = true)
```

Comment:
@BottleRocket-Colin Is the commenting intentional? If not needed, please delete the comments lines.

Reply:
Will do.